backend/formatted_email.py

```diff
@@ -2,39 +2,36 @@
 DATE_KEYWORD = "Date"
 
 class FormattedEmail:
-
   def __init__(self, msg):
     self.msg = msg
     self.formatted_text = ""
     self.formatted_subject = ""
     self.format()
 
   def format(self):
-    lines = self.msg.text.splitlines()    
+    lines = self.msg.text.splitlines()
     self.format_text(lines)
     self.format_subject()
     self.set_foldername(lines)
 
-
   def format_text(self, lines):
     if not lines:
       return
     keyword_line_indices = [i for i in range(len(lines)) if FROM_KEYWORD in lines[i]]
     cut_index = keyword_line_indices[0] if keyword_line_indices else 0
     formatted_text = "\n".join(lines[cut_index:])
     self.formatted_text = formatted_text
-  
+
   def set_foldername(self, lines):
     if not lines:
-      return
+      return self.formatted_subject
```

Comment:
Note: I think we should revisit how we want to name the folders in Dropbox. We can fix this in a follow up diff